packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1290,19 +1295,17 @@ export default class Meetings extends WebexPlugin {
 
       if (meetingInfo) {
         meeting.injectMeetingInfo(meetingInfo, meetingInfoOptions, meetingLookupUrl);
-      } else if (
-        enableUnifiedMeetings &&
-        !isMeetingActive &&
-        useRandomDelayForInfo &&
-        waitingTime > 0
-      ) {
-        meeting.fetchMeetingInfoTimeoutId = setTimeout(
-          () => meeting.fetchMeetingInfo(meetingInfoOptions),
-          waitingTime
-        );
-        meeting.parseMeetingInfo(undefined, destination);
-      } else {
-        await meeting.fetchMeetingInfo(meetingInfoOptions);
+      } else if (!isOne2OneMeeting) {
```

Comment:
Do we not have some parameter within the meeting object that could be used to identify if it is 1:1 meeting instead of introducing new parameter?

Reply:
Currently no. It will be passed by web client based on the video call button clcik